java/src/jmri/jmrix/openlcb/swing/eventtable/EventTablePane.java

```diff
@@ -198,7 +198,7 @@ public void sendRequestEvents(java.awt.event.ActionEvent e) {
         model.loadIdTagEventIDs();
         model.handleTableUpdate(-1, -1);
 
-        final int DELAY = 75; // msec between operations - 64 events at speed
+        final int IDENTIFY_EVENTS_DELAY = 125; // msec between operations - 64 events at speed
```

Comment:
No need to change, but a question.
Is the comment still correct? Or maybe it doesn't matter?
If 64 events per 75 msec, then it seems to me that there would be more events at 125 msec. But I don't use OpenLCB so I don't know what this means.

Reply:
Thanks for checking carefully.  The original calculation was wrong, hence the larger value. (It was based on line speed, but there are short delays between the individual messages)